File: 김경준/1주차/1316.cpp
N = int(input())
cnt = N

for i in range(N):
    word = input()
    for j in range(0, len(word)-1): #인덱스 범위 생성:0부터 단어개수 -1까지
        if word[j] == word[j+1]: #앞 뒤 문자가 같은경우
            pass
        elif word[j] in word[j+1:]: #앞에 그 문자가 있는지 체크
            cnt -= 1
            break

print(cnt)

File: 김진호/1주차/1316.py
N = int(input())

res = 0
for _ in range(N):
    s = list(input())

    alpha = [0] * 26
    flag = True
    i = 0
    for c in s:
        index = ord(c) - ord('a')

        if alpha[index] == 0 or i - alpha[index] == 0:
            alpha[index] = i + 1
            i += 1
        else:
            break
    if i == len(s):
        res += 1
print(res)